src/Masa.Blazor/Components/Select/MSelect.cs

```diff
@@ -516,7 +516,8 @@ protected override void OnInternalValueChange(TValue val)
         {
```

Comment:
Line 243. Because this call is not awaited, execution of the current method continues before the call is completed. Consider applying the 'await' operator to the result of the call.